kolibri/core/assets/src/api-resources/contentNode.js

```diff
@@ -115,11 +115,23 @@ export default new Resource({
     });
     return promise;
   },
-  fetchPopular(getParams) {
-    return this.fetchListCollection('popular', getParams);
+  fetchPopular(params = { popular: true }) {
+    const promise = new ConditionalPromise();
```

Comment:
have we solve the problem with `only` in `ConditionalPromise`? this makes me shiver if we need to use all with several promises (I know `fetchListCollection` also uses `ConditionalPromise` but wondering if we could just use `Promise` to avoid this problem)

Reply:
Yeah - this is a precautionary measure to have a consistent API until we can get rid of `ConditionalPromise` - although your comment here has made me realize a potential way to do that right now.